reference/formal_and_informative/specB03.rst

```diff
@@ -12,4 +12,4 @@
   :start-after: marker_import_units_1
   :end-before: marker_import_units_end
 
-.. include:: ../libcellml/libcellmlB3.rst
+.. todo ../libcellml/libcellmlB3.rst
```

Comment:
What do these do?

Reply:
These are the placeholders for when the 10 so informative sections are
written.  They should be ..include:: directives, only todo so that they're
commented out for now.

Reply:
Thanks!